python/ray/tune/api_v2/dataset_execution_registry.py

```diff
@@ -0,0 +1,24 @@
+from ray.data import Dataset
```

Comment:
Do we need an api_v2 folder? It's a bit confusing since Tuner is closely compatible with the original API.

Recommend getting rid of this folder.